cli/types/cypress.d.ts

```diff
@@ -2548,8 +2548,14 @@ declare namespace Cypress {
     /**
      * A String or Array of glob patterns used to ignore test files that would otherwise be shown in your list of tests. Cypress uses minimatch with the options: {dot: true, matchBase: true}. We suggest using http://globtester.com to test what files would match.
      * @default "*.hot-update.js"
+     * @deprecated use `ignoreSpecPattern` instead
      */
     ignoreTestFiles: string | string[]
+    /**
+     * A String or Array of glob patterns used to ignore test files that would otherwise be shown in your list of tests. Cypress uses minimatch with the options: {dot: true, matchBase: true}. We suggest using http://globtester.com to test what files would match.
+     * @default "*.hot-update.js"
+     */
+    ignoreSpecPattern: string | string[]
```

Comment:
Is the implementation of this going to be a separate PR?

Reply:
Should already be done - it's just a find+replace from `ignoreTestPattern` -> `ignoreSpecPattern`.

Reply:
The pr is wip and here https://github.com/cypress-io/cypress/pull/19345